instructure/canvas-ios
# Offline course sync people list and details

refs: MBL-16766
affects: Student
release note: None.
test plan: 
- Sync People
- Check listed people
- Check user details for current and other users
- Search, filter, and message send button should be disabled in offline

## Checklist
